crates/re_build_tools/src/lib.rs

```diff
@@ -98,7 +112,9 @@ pub fn export_env_vars() {
 }
 
 fn set_env(name: &str, value: &str) {
-    println!("cargo:rustc-env={name}={value}");
+    if should_output_cargo_build_instructions() {
+        println!("cargo:rustc-env={name}={value}");
+    }
```

Comment:
when is `set_env` used? maybe this should fallback to `std::env::set` ?

Reply:
It's used by `re_build_tools::export_env_vars()`.

I think having it fall back is potentially more confusing since it many situations I don't think std::env::set will actually do what the user wants?